coins/ethereum/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "ethereum"
```

Comment:
Mind doing ethereum-serai and filling out the other package fields, as coins/monero or the crypto libs did?

Reply:
done